solc/CommandLineParser.cpp

```diff
@@ -846,42 +872,32 @@ General Information)").c_str(),
 			m_options.output.stopAfter = CompilerStack::State::Parsed;
 	}
 
-	vector<string> const exclusiveModes = {
+	checkMutuallyExclusive({
```

Comment:
I guess the return value was supposed to be used here.

Reply:
Right. Great catch! I'm actually surprised it's not covered by tests. I'll add one.

Reply:
Done.